### DISCUSSION TITLE
Threema account alerts - not gateway

### DISCUSSION THREAD

**Discussion by karalabe** [grafana/grafana#45124] [Alerting]
Hey all,

  I've been wanting to get Threema alerting done via their wire protocol using plain - licensed - accounts, not the Threema Gateway. Been exploring and implementing the Threema messaging protocol towards that end (https://github.com/karalabe/go-threema) and was wondering what the best way would be to do the alerting integration.

  My ideal scenario would be some alerting plugin, but if I interpret the code correctly, the alerting subsystem does not support plugins. I could definitely write up a real Go integration, but I am unsure whether that would be accepted given the Threema Gateway is already in the code.

  Of course, it's always possible to deploy a proxy web service that consumed Grafana alerts via webhooks and translates that into Threema sens, but is seems counterproductive to go through all that complexity + have 1 more service running just to translate Grafana to Threema.

Wondering what the correct path here is and what I could do to advance it. Thanks!

**Comment by karalabe** [comment]
Going down the webhooks path, I can make it work. I just don't like it :)

https://github.com/karalabe/grafana-threema-forwarder

![](https://user-images.githubusercontent.com/129561/153272103-03864925-197d-42db-a40d-0a32e1a7d8b3.jpg)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
